### DISCUSSION TITLE
Mudar de cidade por novas oportunidades

### DISCUSSION THREAD

**Discussion by victorsilent** [frontendbr/forum#728] [Discussão]
Olar pessoas, sou um jovem que está prestes a se formar e estou com esse dilema, atualmente trabalho home office para uma agencia aqui da cidade, porém eu queria trabalhar com coisas mais legais e eu to com o achismo de que o home office não me ajuda a crescer, eu sou o unico front, então não tenho com quem dividir dúvidas e etc, é tudo no papai google. 

Dai vem meu dilema, aqui na cidade não temos um polo bacana de tecnologia, e sempre vejo aqui no repositório de vagas muitas vagas iradas, mas todas em SP/POA/BH e eu tenho uma vontade do caralho de me jogar nessas vagas e dar o gás aprendendo tudo. No meu atual estágio eu me esforço pra fazer as coisas de uma forma bacana, mas junto com a reta final da faculdade fica bem complicado.
O que eu queria mesmo era um feedback de quem ja mudou de cidade devido a emprego, como foi a mudança, como ficaram o primeiro mês, se a empresa deu uma ajuda, se ela levou isso em conta, se é dificil essa parada de alocar em outra cidade e tal, acredito que esse seja meu unico medo mesmo. 
Pra quem não viveu isso ainda mas tem vontade de fazer a mesma coisa, se tiverem um plano ai e quiserem compartilhar, seria massa :B
O meu, até o momento, é juntar uns 10k e mandar bala embora.
Bem, é isso :p

**Comment by bernardodiasc** [comment]
Muda, que quando a gente muda o mundo muda com a gente. 🎶 😎

**Comment by mateusmiguel** [comment]
Fala João! Me mudei de Olímpia pra Campinas há 5 meses. Tava no mesmo dilema que você, trabalhando remoto e vivendo numa cidade tranquila, mas sem área de TI. No meu caso, estava pra casar, então já que ia sair de casa de qualquer forma, me planejei pra vir pra cá. 

Apesar de já ter o emprego 'garantido', o salário não era muito legal e não tinha garantia nenhuma de que haveria demanda pra mim nos próximos meses. Alugamos um apê e viemos na cara e na coragem rs. Com três meses aqui fui pra outra empresa. Pra mim tem sido excelente porque tenho aprendido muito com a galera de lá. Hoje acho anos luz melhor trabalhar junto com o resto do time do que remoto.

Vai na fé!

E ah, a empresa que eu trabalho, inclusive, tem uma cultura bem legal de contratar justamente a galera que ta pra vir/veio de outras cidades.  abs!

**Comment by caiogaspar** [comment]
Eu me formei no final de 2016, trabalhava e morava em Santos/SP, que não é uma cidade que tem campo bom pra tecnologia em geral. Assim que terminei a faculdade comecei a procurar emprego aqui em São Paulo e consegui, valeu muito a pena pra minha carreira, só tenho crescido. Tinha esse medo também, mas depois de um tempo você se acostuma e vê que é o melhor a se fazer!

**Comment by fredmosc** [comment]
Bom, nasci e cresci no RJ, nos últimos anos ficou inviável economicamente aquilo lá, fora todos os outros problemas q vemos todos os dias. Com 32 fui para o interior do RJ e trabalhava remoto, mas tinha esse problema do "eu sozinho" q não permitia feedback e crescimento, fora q me formei em área alheia (Jornalismo, é... nada a ver apesar de já trabalhar com programação desde 1991).

Resolvi me mudar pra Foz do Iguaçu no fim de 2010, não tinha emprego, não conhecia ngm e não tinha diploma na área de TI... cheguei aqui e consegui um trampo em uma agência q dava pra me sustentar sem grandes luxos, o que recebia era bem menor no que no RJ mas o custo/qualidade de vida aqui é bem melhor. Depois de um tempo surgiu a oportunidade de um diploma na área e finalizei o Tecnólogo em Análise e Desenvolvimento pelo IFPR, descolei nesse meio tempo um estágio, depois uma bolsa no Parque Tecnlógico de Itaipu. Fui efetivado agora no meio do ano e to bem feliz.

Todas essas mudanças, eu já tava casado e com uma filha, aqui em Foz casei de novo e tenho mais uma filha. Sem $$ pra segurar não daria, mas tive apoio da minha esposa q segurou a barra um tempo enquanto não entrava muito $$. 

É um resumo do que aconteceu nos últimos 8 anos da minha vida. Veja o que dá pra vc aproveitar. Sucesso!

**Comment by mvfsillva** [comment]
@victorsilent eu sou da terra do dendê (Salvador/BA) e me mudei recentemente (3 meses) e cara eu tinha esse dilema, apresentei o meu TCC e 3 dias depois fui embora, hoje eu estou vivendo o melhor momento da minha carreira,  alguns vão te chamar de louco, outros vão dizer que vc não vai se adaptar! Ignore totalmente essa galera, apenas continue andando sem medo de tropeçar!!

**Comment by calaca** [comment]
Me mudei do interior para a capital do estado (Goiânia) e valeu muito a pena: oportunidades, crescimento profissional, etc. Mas já devo avisar que se você nunca morou sozinho ou não tem o costume de "se virar" em casa, você provavelmente vai sofrer um pouco até se adaptar. Nesse caso, acho que quem se muda com a família tem uma adaptação menos sofrida, até mesmo por questões de solidão.

Mais uma coisa: faça pesquisas de custo de vida para já ir se preparando para eventuais dificuldades financeiras.

**Comment by victorsilent** [comment]
Aaaaa, é por isso que essa comunidade é um amorzinho <3
Muito obrigado pelas experiências galera! Ver essas paradas me da mais vontade de tocar isso pra frente, ja tô quase na metade do valor que estipulei e ano que vem me formo, agora é só esticar os estudos e ano que vem tô prontinho pra cair nesse mundão :p
Vlwzão <3

**Comment by ericmadureira** [comment]
Vou deixar meus 2 centavos: tenho 1 ano e meio de estágio já e ainda não me formei (faltam 2 anos) mas to maluco pra conseguir uma vaga de junior, pena que só aparece nessas cidades que você já citou. As empresas pra onde mandei currículo não dão chance se você mora longe deles, mesmo que a realocação fique por sua conta. 

Eu faço EaD justamente pela flexibilidade no caso de uma vaga aparecer mas ainda não rolou. O plano de juntar dinheiro e se mudar parece bom, é o que to fazendo. Boa sorte na luta aí e quem sabe o pessoal dessa issue se encontre um dia xD

**Comment by mstuttgart** [comment]
Deixando minha colaboração aqui @victorsilent .

A pouco mais de um ano me mudei de uma pequena cidade de MG (Itajubá) para São Paulo (Capital) para iniciar em um novo emprego. Nunca havia saído da minha cidade e nem morado sozinho, mas tive de fazer assim por ter conseguido uma vaga de emprego. Alguns pontos que posso destacar são:

* Antes de se mudar, tenha uma quantia de dinheiro guardada (eu colocaria algo em torno de 5k ou mais), pois com a mudança, procura de local para morar e viagens, você vai gastar um pouco. Quando minha mudei não tive ajuda de custo da empresa, mas como tinha poupado uma grana, consegui me mudar sem problemas.
* Tirando a saudade que sinto da família e da minha noiva, a mudança trouxe apenas coisas positivas. Muito aprendizado, tanto no novo emprego como na vida.
* Seria bacana se mudar com uma vaga já garantida. Pesquise sobre a empresa em que vai trabalhar, verificando se uma empresa bacana pra se trabalhar e etc.

Como o @caiogaspar disse muito bem, no começo você fica com receio e com medo da mudança, mas depois de alguns semanas/meses você se acostuma e vê que tomou a decisão certa. Quando um desafio te traz aquele frio da barriga é porque ele também vai te trazer muito aprendizado. Aprendi isso com a vida. Boa sorte!!
